tests/test_fsx.py

```diff
@@ -0,0 +1,251 @@
+
+        self.assertTrue([t for t in tags['Tags'] if t['Key'] == 'test'])
+
+    def test_fsx_remove_tag_resource(self):
+        session_factory = self.replay_flight_data('test_fsx_remove_tag_resource')
+        p = self.load_policy(
+            {
+                'name': 'test-fsx',
+                'resource': 'fsx',
+                'filters': [
+                    {
+                        'tag:Name': 'test'
+                    }
+                ],
+                'actions': [
+                    {
+                        'type': 'remove-tag',
+                        'tags': [
+                            'maid_status',
+                            'test'
+                        ],
+                    }
+                ]
+            },
+            session_factory=session_factory
+        )
+        resources = p.run()
+        self.assertTrue(len(resources))
+        client = session_factory().client('fsx')
+        tags = client.list_tags_for_resource(ResourceARN=resources[0]['ResourceARN'])
+
+        self.assertFalse([t for t in tags['Tags'] if t['Key'] != 'Name'])
+
+    def test_fsx_mark_for_op_resource(self):
+        session_factory = self.replay_flight_data('test_fsx_mark_for_op_resource')
+        p = self.load_policy(
+            {
+                'name': 'test-fsx',
+                'resource': 'fsx',
+                'filters': [
+                    {
+                        'tag:Name': 'test'
+                    }
+                ],
+                'actions': [
+                    {
+                        'type': 'mark-for-op',
+                        'op': 'tag'
+                    }
+                ]
+            },
+            session_factory=session_factory
+        )
+        resources = p.run()
+        self.assertTrue(len(resources))
+        client = session_factory().client('fsx')
+        tags = client.list_tags_for_resource(ResourceARN=resources[0]['ResourceARN'])
+
+        self.assertTrue([t for t in tags['Tags'] if t['Key'] == 'maid_status'])
+
+    def test_fsx_update_configuration(self):
+        session_factory = self.replay_flight_data('test_fsx_update_configuration')
+        p = self.load_policy(
+            {
+                'name': 'test-update-fsx-configuration',
+                'resource': 'fsx',
+                'filters': [
+                    {
+                        'WindowsConfiguration.AutomaticBackupRetentionDays': 1
+                    }
+                ],
+                'actions': [
+                    {
+                        'type': 'update',
+                        'WindowsConfiguration': {
+                            'AutomaticBackupRetentionDays': 3
+                        }
+                    }
+                ]
+            },
+            session_factory=session_factory
+        )
+        resources = p.run()
+
+        self.assertEqual(len(resources), 1)
+
+        p = self.load_policy(
```

Comment:
please don't use policies to verify policies, use a client and api calls. a policy verification results in slower tests, duplicated recorded data, etc.